### DISCUSSION TITLE
Welcome to getting_badges Discussions!

### DISCUSSION THREAD

**Discussion by rajkishorbgp** [rajkishorbgp/getting_badges#6] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rajkishorbgp** [comment]
How are u?

**Reply by MrRajkishor** [reply]
hmm

**Reply by rajkishorbgp** [reply]
🤔
